src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,52 @@
-export const Tabs = () => {};
+import { useState } from 'react';
+import classnames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const [localSelectedTabId, setLocalSelectedTabId] = useState(selectedTabId);
```

Comment:
do you really need it? Can't be used only `selectedTabId`?

Reply:
This allows the component to maintain its own local state for the selected tab. So yeah, I think it's OK, and if it's possible I would leave it